lib/pg_ha_migrations/safe_statements.rb

```diff
@@ -154,6 +155,10 @@ def unsafe_make_column_not_nullable(table, column, options={}) # options arg is
     end
   end
 
+  # In early versions of Rails 6.0 (fixed in 6.1.4), there is a bug in the SQL
+  # generation when if_not exists is present: https://github.com/rails/rails/pull/41490
+  #
+  # Should we backport a fix in our hacks, raise an error, or do nothing?
```

Comment:
Left a code comment here just as a reminder to myself to add a comment on the PR and start a discussion 😄

Reply:
I think my opinion is to do nothing ™️ 

I don't think it's actively _harmful_, so if a user runs into this bug, they should just bump the patch version of rails, which is generally pretty straightforward. Maybe just add a note in the readme?

Reply:
Yeah, adding a note to the readme would be good. If we wanted to we could catch that specific error and re-raise a message about that, but it's probably not worth it.

Reply:
Added some notes in https://github.com/braintree/pg_ha_migrations/pull/85/commits/dba97ce4350453a84b970f7b0855d2f5f3615f9f